app/src/ui/diff/index.tsx

```diff
@@ -238,6 +243,22 @@ export class Diff extends React.Component<IDiffProps, IDiffState> {
   }
 
   private renderTextDiff(diff: ITextDiff) {
+    if (enableExperimentalDiffViewer() || this.props.showSideBySideDiff) {
```

Comment:
Should this be `(enableExperimentalDiffViewer() || (enableSideBySideDiffs() && this.props.showSideBySideDiff))`?

Reply:
It should be equivalent since it should not be possible to have `showSideBySideDiff` to true when `enableSideBySideDiffs()` returns false, but yes adding a double check won't hurt us